hack/run_test

```diff
@@ -60,16 +60,18 @@ function get_cid() {
 
 function get_container_ip() {
   local id="$1" ; shift
-  docker inspect --format='{{.NetworkSettings.IPAddress}}' $(get_cid "$id")
+  local net=${network:-bridge}
+  docker inspect --format="{{.NetworkSettings.Networks.$net.IPAddress}}" $(get_cid "$id")
 }
 
 function get_ip_from_cid() {
   local cid="$1"; shift
-  docker inspect --format='{{.NetworkSettings.IPAddress}}' $cid
+  local net=${network:-bridge}
```

Comment:
>Are there other calls to get_ip_from_cid in which network is undefined and becomes "bridge"?

Yes, currently the `run_replication_test` test case is not using its own network (as it does not need one) while also calling `get_ip_from_cid`.
Also once `get_container_ip` and `get_ip_from_cid` get de-duplicated more test cases will be using `network` undefined.